recipe/meta.yaml

```diff
@@ -26,34 +13,27 @@ source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
   sha256: bdb8e98145469d58c69ab9f2c9e0bd838c2836a99b5760bc0ebf658623768f52
   patches:
-    - patches/force-protoc-executable.patch
```

Comment:
Again, I would refrain from changing this since the conda-forge recipe was changed only only later versions. See https://github.com/conda-forge/grpc-cpp-feedstock/blob/bf30a933d76a05831518bdd680a7f191bca00926/recipe/meta.yaml for the version that matches ours.

Reply:
I use the same patch from conda-forge recipe of v1.48.1 but refresh it and rename it as conda-forge did patches/0004-force-protoc-executable.patch https://github.com/conda-forge/grpc-cpp-feedstock/blob/ee8b755cf426f6bc895228ebc06b3a43a496c918/recipe/meta.yaml#L25